core/src/main/java/org/apache/accumulo/core/iterators/user/ColumnAgeOffFilter.java

```diff
@@ -44,8 +44,10 @@ public TTLSet(Map<String,String> objectStrings) {
         String column = entry.getKey();
         String ttl = entry.getValue();
```

Comment:
I was thinking about the case of ignoring only whitespace also. Could trim here like the following to do that.

```java
String ttl  = entry.getValue().trim();
```

Reply:
Good idea!  Pushed the `trim`.